### ISSUE TITLE
Not able to move to secondary display

### ISSUE THREAD

**Issue by liidtke** [eltoncezar/Spofy#21]
I can't move Spofy to my secondary display, it goes back to the corner of my primary display.

**Comment by eltoncezar** [comment]
Yes, I'm aware of that. It's a bug in my implementation of the "snap to corners" feature. Need more researching.
